### ISSUE TITLE
customize style is missing

### ISSUE THREAD

**Issue by soroushm** [react-native-community/react-native-tab-view#920]
### Current behaviour

in docs [tabStyle](https://github.com/react-native-community/react-native-tab-view#tabstyle), [labelstyle](https://github.com/react-native-community/react-native-tab-view#labelstyle), and other custom style is exist but in code is missing

### Expected behaviour

expected to customize style when pass as props

### Code sample

```javascript

    <TabView
          navigationState={{ index: tabIndex, routes }}
          renderScene={SceneMap({ Chart, Queue, Transactions })}
          onIndexChange={(tabIndex) => this.setState({ tabIndex })}
          initialLayout={{ width: Dimensions.get('window').width }}
          tabStyle={{backgroundColor: 'red', borderRadius: 50}}
          labelStyle={{backgroundColor: 'blue', borderRadius: 50}}
          indicatorStyle={{backgroundColor: 'green'}}
          contentContainerStyle={{backgroundColor: 'yellow'}}
          style={{ backgroundColor: 'pink',  }}
        />
```

### What have you tried

i will send a pull request

### Your Environment

| software                     | version
| ---------------------------- | -------
| ios or android               | 
| react-native                 | 0.60.4
| react-native-tab-view        | ^2.10.0
| react-native-gesture-handler | ^1.3.0
| react-native-reanimated      | ^1.1.0
| node                         | 12
| npm or yarn                  | yarn install v1.17.3

**Comment by mfentross** [comment]
This is still not fixed in v2.13.0

**Comment by soroushm** [comment]
@mfentross you should use render method for overwrite
```
 <TabView
        navigationState={{ index: tabIndex, routes }}
        renderScene={this.renderScene}
        renderTabBar={(props) => (
          <TabBar
            {...props}
            style={styles.tabBar}
            pressColor="transparent"
            indicatorStyle={styles.indicatorStyle}
            renderLabel={({ route, focused }) => (
              <Typography align="center" color={focused ? 'primary' : 'initial'} text={route.title} />
            )}
          />
        )}
        onIndexChange={(tabIndex) => this.setState({ tabIndex })}
        initialLayout={{ width: width - 32 }}
        style={defaultStyle}
      />

```
